jobs/release-charm.yaml

```diff
@@ -35,6 +35,20 @@
             - 'canonical-kubernetes-elastic'
             - 'kubernetes-calico'
             - 'canonical-kubernetes-canal'
+            - 'kubeflow'
```

Comment:
Make all charms have a namspace:

`kubeflow-charmers/kubeflow` and `containers/kuberenetes-core`